test/parallel/test-stream-readable-async-iterators.js

```diff
@@ -43,8 +43,6 @@ async function tests() {
     });
 
     const iter = Readable.prototype[Symbol.asyncIterator].call(stream);
-    await iter.next();
-    await iter.next();
```

Comment:
This test was modified in the original PR, this PR helps bring it back to what it was.